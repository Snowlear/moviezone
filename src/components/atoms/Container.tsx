import ContainerStyles from "./Container.module.css";

export default function Container({ children, styles }: ContainerProps) {
  return (
    <div
      className={
        ContainerStyles.container + (styles === undefined ? "" : " " + styles)
      }
    >
      {children}
    </div>
  );
}

interface ContainerProps {
  children: any;
  styles?: string;
}
