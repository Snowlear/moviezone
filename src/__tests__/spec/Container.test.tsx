import React from "react";
import renderer from 'react-test-renderer';
import Container from "../../components/atoms/Container";

describe("<Container />", () => {
  it("renders <Container /> component with data.", async () => {
    const tree = renderer
    .create(<Container>Test</Container>)
    .toJSON();
  expect(tree).toMatchSnapshot();
  });
});