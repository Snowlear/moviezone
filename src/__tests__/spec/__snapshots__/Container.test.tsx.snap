// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> renders <Container /> component with data. 1`] = `
<div
  className="undefined"
>
  Test
</div>
`;
